scripts/core/runtime.yml

```diff
@@ -174,3 +174,184 @@ params:
       desc: "[in] pointer to tear down parameters"
 returns:
     - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
+--- #--------------------------------------------------------------------------
```

Comment:
Maybe in future we can move this into it's own `adapter.yml`

Reply:
Made a note to revisit this in #728